src/v2/controllers/FormController.js

```diff
@@ -283,4 +287,30 @@ export default Controller.extend({
     button.toggleClass('link-button-disabled', disabled);
   },
 
+  /**
+    * When "Remember My Username" is enabled, we save the identifier in a cookie
+    * so that the next time the user visits the SIW, the identifier field can be 
+    * pre-filled with this value.
+   */
+  updateIdentifierCookie: function(model) {
+    const formName = model.get('formName');
+
+    // Only update the cookie when the user has successfully identified themselves (i.e. if we're on the
+    // challenge-authenticator form) to avoid incorrect/uneccessary updates.
+    if (formName !== FORMS.CHALLENGE_AUTHENTICATOR) {
```

Comment:
Is this always the next step after a user identifies? A user could have disabled mfa?

Reply:
Would be easier to depend on the identifier form, in case other flows get added too.

Reply:
The intention here is to only update the cookie upon successful authentication, relying on the identifier doesn't work when we're using an identifier first flow. I'm confirming this approach with Ian on this thread: https://github.com/okta/okta-signin-widget/pull/2159#discussion_r693078144